cross/pillow/Makefile

```
@@ -0,0 +1,34 @@
+PKG_NAME = pillow
+PKG_VERS = 11.1.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/p/pillow
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS  = cross/freetype cross/libimagequant cross/libjpeg
+DEPENDS += cross/libtiff cross/libwebp cross/zlib
+
+HOMEPAGE = https://github.com/pyca/cryptography
+COMMENT  = Provide cryptographic recipes and primitives to Python developers
+LICENSE  = BSD/ASL2
+
```

copied from cross/cryptography?
```suggestion
HOMEPAGE = https://python-pillow.github.io/
COMMENT  = Python Imaging Library (Fork).
LICENSE  = MIT-CMU

```